### ISSUE TITLE
A practical path to open education

### ISSUE THREAD

**Issue by semioticrobotic** [open-organization-ambassadors/open-org-educators-guide#18]
Proposed chapter from Dan McGuire:

> This chapter provides the background and thinking that has informed the approach of the Stone Arch Bridge Initiative for Education Resources approach in bringing Open to education organizations. This approach  implements openly licensed content and open pedagogy on open source learning management systems which provide results that can then be written about and further expanded upon in open access journals.

**Comment by engineerteacher** [comment]
Possible questions:

1. Considering the author's description of how his understanding of key concepts of openness evolved, what is your own open source journey? What were the major milestones along the way that influenced your practice?
2. The author points to a relative lack of understanding of open principles, concepts, and terminology in schools that he works with. What is your own experience? If it is similar to that of the author, what specific things can be done to raise its awareness? 
3. There is no question that open education resources and open platforms can improve access for students and teachers to high quality instructional materials and resources, thus closing the chronic equity gap. How are the learning organizations you work closest with taking advantage of such resources and what can you do to encourage them to do more?

**Comment by trncb** [comment]
Some copyedit suggestions:

**Gaining traction**

*1st paragraph:*
From "... free to all of the Minneapolis ..."
to "... free to all the Minneapolis ..."

**Opening up professional development**

*1st paragraph:*
from "mulit-age" to "multi-age"
from "curriculum" to "curricula" (the sentence alludes to many curriculums, therefore the plural is more apt)

*2nd paragraph:*
last sentence: from "curriculum" to "curricula"

*4th paragraph:*
From "... 2016, then, I consulted ..."
To "... 2016, I then consulted ..."

*6th paragraph:"
From "... elementary science curriculum ..."
To "... elementary science curricula ..."

*7th paragraph:"
From "... professional development that encompasses a semester ..."
To "... professional development encompassing a semester ..."

**The Augsburg Hybrid Initiative**

*1st paragraph:*
From "... cohort style ..." to "... cohort-style ..."

*2nd paragraph:*
From "... move all of its adult ..."
To "... move all its adult ..."

**Some lessons from the field**

*2nd paragraph:*

There is reference to LMS, but this is not articulated earlier. It might be worth using the "learning management system" earlier in this paragraph to introduce LMS?
eg. 1st sentence from "... must be used in a learning management system." to "... must be used in a learning management system (LMS)."

*3rd paragraph:*
From "... potential to lead organically to a ..."
To "... potential to organically lead to a ..."

*4th paragraph:*
What is "OER"?

*5th paragraph:*
From "Likewise in higher education ..."
To "Likewise, in higher education ..."

From "... proprietary "homework system" in ..."
To "... proprietary "homework systems" in ..."

*6th paragraph:*
"... and wifi-enabled devices."
What do wifi enabled devices have to do with the chapter. I don't think there's any mention of them before or after. Might be good add a handful of words as to why this is relevant to the chapter?

*9th paragraph:*
From "Using openly licensed curriculum with ..."
To "Using openly licensed curricula with ..."

*10th paragraph:*
From "... implementing openly licensed curriculum with ..."
To "... implementing openly licensed curricula with ..."
